samtranslator/translator/translator.py

```diff
@@ -39,14 +43,14 @@
 class Translator:
     """Translates SAM templates into CloudFormation templates"""
 
-    def __init__(  # type: ignore[no-untyped-def]
+    def __init__(
         self,
-        managed_policy_map,
-        sam_parser,
-        plugins=None,
-        boto_session=None,
-        metrics=None,
-    ):
+        managed_policy_map: Optional[Dict[str, str]],
```

Comment:
Is `managed_policy_map` optional? Shouldn't it have `= None` at the end if it is

Reply:
Optional argument doesn't have to include a default value.

Reply:
`Optional[T]` means `Union[None, T]`, not related to whether it's a required argument. I know it's confusing.